manuscript/060_Statement_First_Programming.md

```diff
@@ -124,10 +124,9 @@ public class XmlConfigurationSpecification
   XmlConfiguration config = new XmlConfiguration(xmlFixtureString);
   
   //...
-  //...
 ```
 
-What it does is to set up an object used by every Statement. So, each Statement uses a�`config` object initialized with the same `xmlConfiguration` string value. The string is already pretty huge and messy, since it was made to contain what is required by all existing Statements. You need to write tests for is a�little corner case that does not need all this crap that is inside this string. So, you decide to start fresh and create a�separate object of `XmlConfiguration` class with your own, minimal string. Your Statement begins like this:
+What it does is to set up an object used by every Statement. So, each Statement uses a `config` object initialized with the same `xmlConfiguration` string value. The string is already pretty large and messy, since it contains all what is required by the existing Statements. You need to write tests for a little corner case that does not need all this crap inside this string. So, you decide to start afresh and create a separate object of the `XmlConfiguration` class with your own, minimal string. Your Statement begins like this:
```

Comment:
"contains all **what** is required by the existing Statements" - would it be better to say "contains all **that** is required by the existing Statements"